src/test/java/org/openrewrite/java/migrate/SunNetSslPackageUnavailableTest.java

```diff
@@ -16,15 +16,12 @@
 package org.openrewrite.java.migrate;
 
 import org.junit.jupiter.api.Test;
-import org.openrewrite.DocumentExample;
```

Comment:
This means there's no happy flow example?

Reply:
Good catch, thanks!